docs/index.md

```diff
@@ -42,53 +40,50 @@ See [Building with Vanilla](/building-vanilla) and [Customising Vanilla](/custom
   </div>
 </div>
 
-<hr class="is-deep">
+<br>
 
-<div class="u-equal-height">
+<div class="row u-equal-height">
   <div class="col-12">
     <h3>Download</h3>
-    <p>Download the latest version of Vanilla from GitHub.</p>
+    <span>Download the latest version of Vanilla from GitHub.</span>
     <a href="https://github.com/canonical-web-and-design/vanilla-framework/archive/v2.3.0.zip" class="p-button--positive">Download v2.3.0</a>
   </div>
 </div>
 
-<hr class="is-deep">
+<br>
 
 <div class="row">
+<h3>Release notes</h3>
   <div class="col-6">
-    <h3>What's new</h3>
-    <ul class="p-list">
-      <li class="p-list__item--deep">
-        <a href="https://github.com/canonical-web-and-design/vanilla-framework/releases/tag/v2.3.0">Release notes: v2.3.0</a>
-      </li>
-      <li class="p-list__item--deep">
-        <a href="https://github.com/canonical-web-and-design/vanilla-framework/releases/tag/v2.2.0">Release notes: v2.2.0</a>
-      </li>
-      <li class="p-list__item--deep">
-        <a href="https://github.com/canonical-web-and-design/vanilla-framework/releases/tag/v2.1.0">Release notes: v2.1.0</a>
-      </li>
+    <ul class="p-list--divided">
```

Comment:
Rather than two lists, this should be a split list. It looks the same as two lists at large screen but doesn't have the same problems at smaller screens.

https://docs.vanillaframework.io/patterns/lists/#split

Reply:
Ah ok, will try it with our split list component.